proposals/0344-distributed-actor-runtime.md

```diff
@@ -1444,7 +1444,7 @@ As we can see, this thunk is "just" taking care of converting the heterogeneous
 
 The thunk again uses the indirect return, so we can avoid any kind of implicit existential boxing even on those layers. Errors are always returned indirectly, so we do not need to do it explicitly.
 
-#### Recipient: Collecting result/error from the Invocation
+#### Recipient: Collecting result/error from the invocation
```

Comment:
I think either we have to capitalize `invocation` here or make it `an` invocation instead of `the`.

Reply:
Every title uses the plural form "invocations" now👀